test/integration/driver-virtualbox.bats

```diff
@@ -274,3 +274,31 @@ findMemorySize() {
   run rm -rf $MACHINE_STORAGE_PATH
```

Comment:
@tombee  i think you need to move this test (which isn't really a test, it's a test cleanup) to the end of the file. It's going to remove the temporary machine store that we've created in $MACHINE_STORAGE_PATH (/tmp/ on the mac) and we don't want to do that until we've run all the tests (otherwise it will pollute our production store). **So take lines 273-276 and move them after line 303.**